spec/teaspoon/generators/install_generator_spec.rb

```diff
@@ -16,6 +16,13 @@
     expect(described_class.desc).to eq("Installs the Teaspoon initializer into your application.")
   end
 
+  def rails_6_option
+    return unless Rails::VERSION::MAJOR >= 6
+      "
```

Comment:
I could use a `\n` instead and it will silence hound. However, the indentation will still be confusing since `[--skip-namespace]` starts with 4 spaces.

Reply:
oh, I get it all now.. I thought this was in the generator, it's just in the spec.. my bad. yeah, this is fine.